urllib3/_collections.py

```diff
@@ -305,13 +306,22 @@ def from_httplib(cls, message):  # Python 2
         # python2.7 does not expose a proper API for exporting multiheaders
         # efficiently. This function re-reads raw lines from the message
         # object and extracts the multiheaders properly.
+        obs_fold_continued_leaders = (' ', '\t',)
```

Comment:
Nit: trailing comma isn't needed.

Reply:
Definitely not needed; I've picked it up as a stylistic thing, though, with tuples; they require a trailing comma to build a 1-tuple, so I've just been doing it consistently.

Reply:
I am unsure whether this consistency is a hobgoblin of mine or not, so opinions on that topic are quite welcome. 😉

Reply:
It's just the only thing that stood out to me in this code. :) I think it looks great either way. :+1:

Reply:
Given that it's not the style of the project, I'd rather we either introduce a check for this to ensure it's consistent across the project or drop it.

Reply:
That's fair. Given that I don't want to blow up the scope of this PR, I'll remove the trailing comma for now and revisit the style later on.